yt_dlp/extractor/facebook.py

```diff
@@ -481,6 +484,28 @@ def _extract_from_url(self, url, video_id):
         webpage = self._download_webpage(
             url.replace('://m.facebook.com/', '://www.facebook.com/'), video_id)
 
+        sjs_data = [self._parse_json(j, video_id, fatal=False) for j in re.findall(
+            r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
+
+        cookies = self._get_cookies(url)
+        if all(k in cookies for k in ["c_user", "xs"]):
+            if get_first(sjs_data, (
+                    'require', ..., ..., ..., '__bbox', 'define',
+                    lambda _, v: 'CurrentUserInitialData' in v, ..., 'ACCOUNT_ID'), default="0") == "0":
+                raise ExtractorError('Failed to login with provided data.', expected=True)
+            if any(content in webpage for content in ['180 days left to appeal', 'suspended your account']):
+                raise ExtractorError('Login account is suspended.', expected=True)
```

Comment:
no punctuation at the end of `ExtractorError` msgs
```suggestion
                raise ExtractorError('Your account is suspended', expected=True)
```